File: php-jsonparser/parser.php
<?php
$jsonsDir="data/jsons";
$jsonUrl='https://sedeaplicaciones.minetur.gob.es/ServiciosRESTCarburantes/PreciosCarburantes/EstacionesTerrestres/';

if (!is_dir($jsonsDir)) {
    if(!mkdir($jsonsDir, 0755, true)) {
        die('Fallo al crear las carpetas...');
    }
}
$jsonOriginal = file_get_contents($jsonUrl);

$jsonDecoded = json_decode($jsonOriginal, true);
foreach ( $jsonDecoded['ListaEESSPrecio']  as $eess ) {
    $filename='es::'. $eess['IDEESS'];
    file_put_contents( "$jsonsDir/$filename.json", json_encode($eess));
}
echo "Look for jsons in $jsonsDir. Zip'em and use cbdocloader.";
echo PHP_EOL;
?>

File: python/parser.py
import json
import sys
import urllib2
import os
import errno

jsonsDir = "data/jsons"
jsonUrl ='https://sedeaplicaciones.minetur.gob.es/ServiciosRESTCarburantes/PreciosCarburantes/EstacionesTerrestres/'
prepend = 'eess::'
# creamos el directorio de destino 
try:
    os.makedirs(jsonsDir)
except OSError as exc:  # Python >2.5
    if exc.errno == errno.EEXIST and os.path.isdir(path):
        pass
    else:
        raise

# eessUrl = sys.argv[1]
f = urllib2.urlopen(jsonUrl)
data = json.load(f)
listaEESS = data['ListaEESSPrecio']
for i in range(0, len(listaEESS)):
   eessid = listaEESS[i]['IDEESS']
   outfile = open(jsonsDir + '/' + prepend + str(eessid) + '.json', 'w')
   json.dump(listaEESS[i], outfile)
   outfile.close()


